jmeter-java-dsl/pom.xml

```diff
@@ -88,6 +88,33 @@
       <!-- same version as JMeter dependency -->
       <version>3.6.1</version>
     </dependency>
+
+    <dependency>
```

Comment:
it seems this pr is not rebased with latest changes from master, since all these dependencies have been moved to parent project